packages/design-picker/src/utils/designs.ts

```diff
@@ -44,13 +44,18 @@ export const getDesignPreviewUrl = (
 	// The preview url is sometimes used in a `background-image: url()` CSS rule and unescaped
 	// parentheses in the URL break it. `addQueryArgs` and `encodeURIComponent` don't escape
 	// parentheses so we've got to do it ourselves.
-	const siteTitle = options.site_title || design.title;
+	const siteTitle = options.site_title || preview_data?.site_title || design.title;
```

Comment:
I can't actually get it to use the site_title I give it in onboarding...but then I can't get it to do that on staging either, maybe I'm missing something.

Reply:
Apparently we only use the custom title/tagline in the big preview screen after selecting a design:

https://github.com/Automattic/wp-calypso/blob/6cc7601ed7e4aab61c12ff9774ade8499805a859/client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx#L612-L619

<img width="1269" alt="Screenshot 2023-04-19 at 12 19 38" src="https://user-images.githubusercontent.com/1233880/233045437-af107e7b-e60f-4362-8b49-c1aca259ddb1.png">